<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowBook</class>
 <widget class="QWidget" name="BorrowBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="introduce">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>93</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="name1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>201</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="time1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="time2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="time3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>82</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="RTB1">
      <property name="text">
       <string>归还</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RTB2">
      <property name="text">
       <string>归还</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RTB3">
      <property name="text">
       <string>归还</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
